thingif/src/main/java/com/kii/thingif/temporal/base/BaseEquals.java

```diff
@@ -0,0 +1,6 @@
+package com.kii.thingif.temporal.base;
+
+public interface BaseEquals {
```

Comment:
That is possible but i prefer to start from interface.
There's not much logic to be implemented.